### ISSUE TITLE
voot is not working

### ISSUE THREAD

**Issue by fisby0302** [denobisipsis/adownloader#194]
Hello
Please check the link is not working. Everything worked fine yesterday...
https://www.voot.com/shows/kaisi-yeh-yaariaan-s01/1/361251/tips-&-tricks/365643 
![image](https://user-images.githubusercontent.com/59845766/185808441-058103de-f187-4e67-aae3-a58678ae9d72.png)

**Comment by denobisipsis** [comment]
Use US vpn

**Comment by fisby0302** [comment]
Do I need to download a separate program or is it enough to set the settings in paragraphs 6 and 7?

**Comment by denobisipsis** [comment]
> Do I need to download a separate program or is it enough to set the settings in paragraphs 6 and 7?

No, use external VPN only if you have an account (fill it in voot-credentials). Now, if you havent one the program get the streams but it does not work always ;)
